extensions/graph/pkg/service/v0/service.go

```diff
@@ -89,6 +91,20 @@ func NewService(opts ...Option) Service {
 				}
 			}
 
+			if options.Config.Identity.LDAP.CACert != "" {
+				if tlsConf == nil {
+					tlsConf = &tls.Config{}
+				}
+				certs := x509.NewCertPool()
+				pemData, err := ioutil.ReadFile(options.Config.Identity.LDAP.CACert)
```

Comment:
When using this with the defaults, this is racy. The default cert is generated when starting the `idm` service. Which happens in parallel to starting the `graph` service. I guess we should do here what `idp` does: https://github.com/owncloud/ocis/blob/master/extensions/idp/pkg/service/v0/service.go#L47

Reply:
Yeah, you're right. I'll implement that behavior. :+1: